### DISCUSSION TITLE
Clarification on what addons should be added here

### DISCUSSION THREAD

**Discussion by loganmcbroom** [LuaLS/LLS-Addons#95] [General]
I have an LLS addon for what could easily be called a personal project, but it is also an open source software package I am distributing to others. I am wondering if there is a well defined cutoff for what makes something popular enough to add? It would be great to make getting the addon installed easier for users, as the software is painful to use without it.

Thanks =)

**Comment by carsakiller** [comment]
I have been wanting to make it easy for users to install any addon of their choosing, in any editor of their choosing, not just the ones in LLS-Addons using VS Code. I intend to move the install logic into the language server (I would prefer this, but need help) or a different CLI tool, but until then, LLS-Addons is the current solution. Because it is a git repo, and we are kind of abusing git's clone feature, I have been trying to keep the number of addons in the repo small. This is the reason for the "no personal projects" rule. If you are distributing your project for others to use, though, I think that is fine to be included 🙂.

**Reply by loganmcbroom** [reply]
Woohoo thanks =) 

I can't say I know enough about the current system to know how to improve it, but I will try looking around the language server and thinking about it. Optimally, how do you envision the addon installation process working? An editor agnostic gui tool? Or just being able to call "lls install http://github.com/repo" from terminal would be pretty convenient.

**Reply by carsakiller** [reply]
I've been thinking about it for a long while. I [posted my current idea as a discussion](https://github.com/LuaLS/lua-language-server/discussions/2408) if you'd like to take a look and provide some feedback.
